db/schema.rb

```diff
@@ -364,6 +375,8 @@
   add_foreign_key "course_options", "sites", on_delete: :cascade
   add_foreign_key "courses", "providers"
   add_foreign_key "emails", "application_forms", on_delete: :cascade
+  add_foreign_key "notes", "application_choices"
```

Comment:
I was wrong about the default, we need to add something to make this a `on_delete: :cascade`.

Reply:
Do yo mean adding this as in the `ProviderUser` / `ApplicationChoice` model or to the DB as a whole?